go.mod

```diff
@@ -36,4 +37,4 @@ require (
 	golang.org/x/crypto v0.17.0 // indirect
 )
 
-// replace github.com/invopop/gobl => ../gobl
+// replace github.com/invopop/gobl => github.com/noplisu/gobl v0.0.0-20230327142000-57a48df89539
```

Comment:
[idea] Once the new changes are implemented in invopop/gobl, this change won't be needed, right? Perhaps it makes sense for it to be in a separate commit. Then we can drop it easily.